src/main/webapp/script.js

```diff
@@ -162,6 +163,32 @@ function isUserRegistered() {
   })
 }
 
+/**
+ * Gets majors from database and appends them to select container in mentor and mentee signup
+ * forms.
+ */
+function fetchMajor() {
```

Comment:
nit: I think it makes more sense for this to be "fetchMajors"

Reply:
Done!